### PR TITLE
Added Meta Tag & Meta Image to Index HTML

### PR DESCRIPTION
**Pull request by testerslol123**
Added Meta Tag for sharing to media social like Facebook & Twitter.

Can help check if the link is correct or no?

And can I request the hacktoberfest & hacktoberfest-accepted?
Thank you.

### CODE REVIEW

**Comment by uzair-ashraf** [comment]
Hello! Thanks for contributing, we just pushed an update recently that changes the index.html file, its also in the src folder now, go ahead and take a look.

**Comment by testerslol123** [comment]
@uzair-ashraf Hi, I just redo the revise version of this.
Currently I added index html some meta tag and image for them. 
You can check into the latest commit I do.

Thank you :)

**Comment by uzair-ashraf** [comment]
@testerslol123 Hey dude, can you remove the commented code and send some screenshots with the test?

**Comment by testerslol123** [comment]
sure uzair. Would do it. wait ya

**Comment by testerslol123** [comment]
@uzair-ashraf hello uzair, I just do the test. And screenshot it at below.
https://imgur.com/Xb3fqhb

By removing commented code, what is that mean?
Which one I need to remove?

**Comment by uzair-ashraf** [comment]
@testerslol123 sorry dude! There was a commented block for metaImage

And as for the tests we run cypress now , I'll open a staging branch so I can review it on my end as well

**Comment by testerslol123** [comment]
@uzair-ashraf My bad, I just saw there's update in upstream here. 
I just git pull them and update it in my localhost.
Now I just push the latest with cypress image included.
1. https://ibb.co/Dp5Gybq
2. https://ibb.co/SRX0c3P
3. https://ibb.co/zRHpYsL
4. https://ibb.co/Vg7BNzF

Hope if there's anything that I can help with.

File: webpack.config.js [hunk 1]
```diff
@@ -16,9 +16,13 @@ module.exports = {
...
     new HTMLWebpackPlugin({
       hash: true,
-      title: 'Genshin Impact Wish Sim',
+      title: 'Genshin Impact Wish Simulator',
       template: `${__dirname}/src/index.html`,
       filename: `${__dirname}/dist/index.html`,
+      metaName: 'Genshin Impact Wish Simulator',
+      metaDesc: 'A simulator to let your wish come true. Test your luck here',
+      metaUrl: 'https://uzair-ashraf.github.io/genshin-impact-wish-simulator',
+      metaImage: `${__dirname}/src/assets/images/banners/sparkling-steps.png`
```
Comment (uzair-ashraf) [inline webpack.config.js:25]: Url is different now, and we should use the banner for the current character event wish

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]
Sorry for my lagged response too, got a couple small changes for ya

File: webpack.config.js (hunk 1 above)
Comment (testerslol123) [reply webpack.config.js:25]: sure, let me changed them. Sorry for long response.
By today, ganyu banner only out, do you have updated image? So I can help update on them?

Comment (uzair-ashraf) [reply webpack.config.js:25]: Ganyu banner is in repo

**Pull request closed without merging**